pkcs7/pkcs7-verify.c

```diff
@@ -26,11 +26,16 @@
 
 #ifdef HAVE_PKCS7
 
+static const char* pkcs7SignedDer = "signed.p7b"; /* DER */
```

Comment:
This line is not used. It would be cool if you could compare (memcmp) that this file matches the converted output. Also would be nice to add test for wc_DerToPem().